src/Fields/broadcast.jl

```diff
@@ -32,15 +32,38 @@ Base.Broadcast.broadcastable(field::Field) = field
 Base.eltype(bc::Base.Broadcast.Broadcasted{<:AbstractFieldStyle}) =
     Base.Broadcast.combine_eltypes(bc.f, bc.args)
 
+# _first: recursively get the first element
+_first_data_layout(data::DataLayouts.VF) = data[1]
+_first_data_layout(data::DataLayouts.DataF) = data[]
+_first(x::Real) = x
+_first(x::Geometry.LocalGeometry) = x
+_first(data::DataLayouts.VF) = data[]
+_first(field::Field) = _first_data_layout(field_values(column(field, 1, 1, 1)))
+_first(space::Spaces.AbstractSpace) =
+    _first_data_layout(field_values(column(space, 1, 1, 1)))
+_first(bc::Base.Broadcast.Broadcasted) = _first.(bc.args) # Is this case necessary?
+_first(x::Base.RefValue{T}) where {T} = x
+unref(x::Ref{T}) where {T} = x.x
+unref(T) = T
```

Comment:
Do you still need this?

Reply:
Yes, we do when we pass `FT` into the function.

Reply:
Otherwise we get `DataType` somewhere